src/Expensify.tsx

```diff
@@ -206,18 +205,14 @@ function Expensify() {
             // We use custom deeplink handler in setup/hybridApp
             if (CONFIG.IS_HYBRID_APP) {
                 Report.doneCheckingPublicRoom();
```

Comment:
I think this comment is no longer valid here, right?
Should we remove `Report.doneCheckingPublicRoom();` then? 🤔

Reply:
Yeah good point.